hcatalog/webhcat/svr/src/main/java/org/apache/hive/hcatalog/templeton/tool/HDFSCleanup.java

```diff
@@ -113,8 +112,7 @@ public void run() {
         }
 
         long sleepMillis = (long) (Math.random() * interval);
-        LOG.info("Next execution: " + new Date(new Date().getTime()
-                             + sleepMillis));
+        LOG.info("Next execution: " + (System.currentTimeMillis() + sleepMillis));
```

Comment:
This changes the behavior because it previously would print a DATE, now it's just printing a number.